src/pages/Home.vue

```diff
@@ -64,9 +64,9 @@
 </template>
 
 <script>
-import VueHeader from "@/components/VueHeader.vue";
-import VueTable from "@/components/VueTable.vue";
-import VueGraphic from "../components/VueGraphic.vue";
```

Comment:
The name is redundant, we know is a `vue` file, so don't add unnecessary prefixes.